### PR TITLE
Add oclif plugin support

### PR DESCRIPTION
**Pull request by ponelat**
Adds oclif/plugins plugin

## Proposed Changes

  - Adds `@oclif/plugins` which supports installing 3rd party plugins
  - Add documentation and point to an example plugin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -273,6 +274,41 @@ OPTIONS
...
+$ swaggerhub plugins:install <github-url>
+```
+
+To list other options related to plugins
+
+```sh-session
+$ swaggerhub plugins --help
+```
+
+An example plugin used for fetching populat JSON Schema files, can be found here:  [https://github.com/ponelat/swaggerhub-cli-plugin-schema](https://github.com/ponelat/swaggerhub-cli-plugin-schema)
```
Comment (barryadk) [inline README.md:293]: *populat*

File: package-lock.json [hunk 2]
```diff
@@ -3928,6 +4014,11 @@
         "camelcase": "^5.0.0",
         "decamelize": "^1.2.0"
       }
+    },
+    "yarn": {
```
Comment (barryadk) [inline package-lock.json:4018]: Any reason for including *yarn*?

Comment (ponelat) [reply package-lock.json:4018]: Sadly it's part of `@oclif/plugins`. It's overkill in my opinion, but it's upstream.

**Comment by ponelat** [comment]
@Rody-Kirwan @barryadk who has merging powers?

**Comment by barryadk** [comment]
Could you remove the 'STEAM' ID from the title before you merge since it doesn't make sense in this open source repo?

**Pull request merged**